.github/workflows/pytest.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
         matrix:
             os: [macos-latest, windows-latest, ubuntu-latest]
-            python-version: ['3.9', '3.8', '3.7']
+            python-version: ['3.9', '3.8']
```

Comment:
Perhaps support 3.10?

Reply:
Seems like FastText does not support 3.10: 
https://github.com/facebookresearch/fastText/issues/1274